transport/impls/ws/connection.ts

```diff
@@ -19,15 +57,15 @@ export class WebSocketConnection extends Connection {
   }
 
   addCloseListener(cb: () => void): void {
-    this.ws.onclose = cb;
+    this.closecb = cb;
```

Comment:
```suggestion
    this.closeCb = cb;
```

for casing consistency

Reply:
I thought this is Go :P

Reply:
i wonder why event listeners like onclose or onbeforeunload in web are all lower case lol

Reply:
yeah actually have no idea lol just thought about this too (no strong pref tbh)